packages/terra-show-hide/src/ShowHideFocuser.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { forwardRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNamesBind from 'classnames/bind';
+import { v4 as uuidv4 } from 'uuid';
+import styles from './ShowHide.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Text that precedes the focusable portion.
+   */
+  focusableText: PropTypes.string.isRequired,
+  /**
+   * The portion of the text that gets focus once the hidden portion is disclosed to be announced by assistive technologies
+   */
+  prefix: PropTypes.string,
+};
+
+/**
+ * @param str - string value
+ * @returns a trimmed string with trailing white space added
+ */
+const addTrailingWhiteSpace = (str) => `${str.trim()} `;
+
+/**
+ * @param str - string value
+ * @returns a trimmed string with preceding whitespace added
+ */
+const addHeadingWhiteSpace = (str) => ` ${str.trim()}`;
+
+/**
+ * This method splits the string in two portions without splitting words. The first (focusable) portion's length should not exceed max length for assistive technologies in the text mode being able to use Arrow Up?Down correctly.
+ * @param string - the text
+ * @returns the array of strings, where the first value is short focusable string and the second value is the rest of the string (if such exists)
+ */
+const truncate = (string) => {
+  if (!string) {
+    return null;
+  }
+  const text = string.trim();
+  const maxLength = 155;
```

Comment:
Is this value defined somewhere as the maximum for assistive technologies?  I am not seeing this value when doing a Google search.

Reply:
It is not documented anywhere, I had to experiment with JAWS to determine that number.